### DISCUSSION TITLE
The ability to overwrite spritesheets without overwriting entire XMLs

### DISCUSSION THREAD

**Discussion by pantherpusher** [FakeFishGames/Barotrauma#12076] [Ideas and suggestions]
I just wanna be able to replace the sprites for the human without needing to overwrite all the human XMLs and thus make it incompatible with everything else that touches that part of the code, I know it's gonna mean a lot of work, I've seen how sprites are tied to XML and how sourcerects are done and shit but frankly I think the system is flawed as is and sprite sheets would be better served able to have their own dedicated XML file defining which sprite is which. Instead of the item saying who owns what plot of PNG. It'd be best implemented in a way that doesn't render all previous mods instantly incompatible since it'd be a major structure change. I think it'd be a worthwhile change and open up doors that are otherwise bricked over and removed from the original floorplan.

**Comment by ValerieSteel** [comment]
Agreed. I just want to have a more stylized look for my character without going full moe-waifu-sugoi-whateverthefuck, and this would make having that much easier.

**Comment by Nebulisk** [comment]
It's convenient for everyone, it might be a hassle to do, but preferably please!

**Comment by BiffW** [comment]
Just hope they absorb the xpath patch into the base game. You can partially override anything with that, not just sprites. Thus, mods that override different part of the same item will be compatible without the need of extra compatibility patch.
